recipe/meta.yaml

```diff
@@ -10,29 +10,29 @@ source:
 
 build:
   skip: True  # [py<37]
-  number: 0
+  number: 1
   script:
-    - python -c '__import__("os").unlink("pyproject.toml")'
-    - {{ PYTHON }} -m pip install . -vv
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - pip
     - wheel
-    - poetry
+    - poetry-core >=1.0.0
   run:
     - python
-    - typing_extensions >=4.4  # [py<310]
+    - typing-extensions >=4.7.1  # [py<38]
+    - typing-extensions >=4.9    # [py>=38]
```

Comment:
typing-extensions are required only for `py<3.8`, see https://github.com/graphql-python/graphql-core/blob/430ee891a52d239adadf133d873fdca2a3fe009b/pyproject.toml#L47-L49